guides/quickstart.md

```diff
@@ -17,15 +17,61 @@ In the following script the basic setup of the database is executed before ZITAD
 
 You can connect to [ZITADEL on localhost:4200](http://localhost:4200) after the frontend compiled  successfully. Initially it takes several minutes to start all containers.
 
+<a name="compose-services"></a>
 ```bash
-COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 \
-&& docker compose -f ./build/local/docker-compose-local.yml --profile backend --profile frontend up
+$ COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker compose -f ./build/local/docker-compose-local.yml --profile backend --profile frontend up --build. Setting everything up takes about 5 minutes.
 ```
 
-For a more detailed guide take a look at the [development guide](./development.md)
+## Developing ZITADEL
+
+Instead of the profiles backend and frondend as described [above](#compose-services), use the profile e2e and detach from containers. Setting everything up takes about 15 minutes. This also initializes data needed by Cypress end-to-end tests. 
```

Comment:
```suggestion
Instead of the profiles backend and frondend as described [above](#compose-services), use the profile e2e and detach from containers. The e2e profile includes the backend and frontend profiles plus adds the e2e-setup service which prepares the environment for end-to-end tests. Setting everything up takes about 15 minutes. 
```